app/src/main/res/values/strings.xml

```diff
@@ -110,6 +110,16 @@
     <string name="system">System</string>
     <string name="theme_desc">The theme used by the application</string>
 
+    <!-- buttons -->
+    <string name="confirm">Confirm</string>
+    <string name="cancel">Cancel</string>
+    <string name="claim">Claim</string>
+
+    <!-- team progress -->
+    <string name="claim_confirmation_alert_title">Confirm claim</string>
+    <string name="claim_confirmation_alert_text">Are you sure you want to claim this challenge ?</string>
```

Comment:
In english, there is no space between the last word and the question mark if I am not mistaken, but, really minor.

Reply:
Yes you're right